packages/vitest/src/node/core.ts

```diff
@@ -163,7 +163,17 @@ export class Vitest {
     const projects = await this.resolveWorkspace(cliOptions)
     this.projects = projects
     this.resolvedProjects = projects
-    const filteredProjects = toArray(resolved.project)
+    const filteredProjects = toArray(resolved.project).reduce((acc, s) => {
```

Comment:
nit: filtering projects twice looks a bit odd. How about doing this in one-go like this?

```tsx
      const patterns = filteredProjects.map(s => wildcardPatternToRegExp(s));
      this.projects = this.projects.filter(project =>
        patterns.some(pattern =>
          project.getName().match(pattern)
        )
      );
```

Reply:
Thanks! Already updated.